src/warnet/lnnode.py

```diff
@@ -0,0 +1,115 @@
+
+import docker
+import json
+import os
+from docker.models.containers import Container
+from warnet.client import get_file_from_container
+from warnet.utils import (
+    exponential_backoff,
+    generate_ipv4_addr
+)
+
+CONTAINER_PREFIX_LN = "ln"
+
+class LNNode:
```

Comment:
Seems like it might behoove us to make `LNNode` another abstract base class, which can be overridden by an LND node, CLN node or other future node types?

Reply:
Yeah thats the plan. Just haven;t gotten to CLN yet